dotnet/src/SemanticKernel.Abstractions/Contents/AudioContent.cs

```diff
@@ -16,7 +16,15 @@ public class AudioContent : KernelContent
     /// <summary>
     /// The audio binary data.
```

Comment:
I noticed you did the same for `ImageContent`, should we update it here as well?
```suggestion
    /// The audio data.
```